back/engines/commercial/project_folders/app/models/project_folders/folder.rb

```diff
@@ -61,36 +62,36 @@ def admin_publication_must_exist
 
     def generate_slug
       slug_service = SlugService.new
```

Comment:
Just wondering: how many times is this method expected to be called? From the name, I would expect only once. From the code, it seems like it could be called multiple times, because of the presence of `||=`.

If it can be called multiple times, isn't it better then to write it as:
```suggestion
      self.slug ||= SlugService.new.generate_slug self, title_multiloc.values.first
```
so that the creation of the SlugService instance is avoided?